wp-content/themes/core/blocks/tribe/terms/style.pcss

```diff
@@ -52,3 +52,14 @@
 		@mixin t-tag;
 	}
 }
+
+/* -----------------------------------------------------------------------
+ * Terms Block - Empty Terms List
+ * ----------------------------------------------------------------------- */
+
+.wp-block-tribe-terms__empty-msg {
+
+	.editor-styles-wrapper & {
+		color: var(--color-neutral-60);
+	}
+}
```

Comment:
❓ Should we add the `t-tag` mixin here so the styling shows correctly depending on the style chosen or is it ok that this displays differently?

Reply:
Good catch. Updated.